.github/workflows/build.yaml

```diff
@@ -98,3 +98,33 @@ jobs:
           tag: ${{ github.ref }}
           overwrite: true
           file_glob: true
+
+  release-checksums:
+    name: Release Artifact Checksums
+    needs: build-arch
+    if: github.event_name == 'release' && github.event.action == 'published'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Download Release Artifacts
+        uses: robinraju/release-downloader@v1.9
+        with:
+          tag: ${{ github.ref_name }}
+          fileName: "z*"
+      - name: Generate checksum
+        uses: jmgilman/actions-generate-checksum@v1
+        with:
+          patterns: z*
+          method: sha256
+          output: checksums.sha256.txt
+      - name: Add wildcard character prefix to filenames in checksum file
+        run: sed -i 's! ! \*!g' checksums.sha256.txt
```

Comment:
Why do we need this step?

Reply:
Hi @andaaron, by default, the checksum action adds the filename to the result file without a leading `*`. For example:
```
9d8543befd1a1fc57c6a08ce0054cfbc2f3999995636a367996a2b698110d0f6 zot-darwin-amd64
84d1e2743df9a7f44162e8eb677983907724232890f45603b2eaaba95ac5eed1 zb-linux-arm64
```
This works great on Linux and `sha256sum` is able to verify the hashes.
However, on MacOS, I observed that `shasum` fails with an error: 
`shasum: checksums.sha256.txt: no properly formatted SHA checksum lines found`.

Adding the asterisk as a prefix to the name allows `shasum` to verify the checksums on MacOS.
Here's a reference to a Stackoverflow answer which gave me the hint for the asterisk https://stackoverflow.com/questions/70502738/sha256sum-standard-input-no-properly-formatted-checksum-lines-found

Reply:
Here's a better example:
On Linux:
```
echo "a39bc62c6175b85d9610781bd8811fa13473c9b5f32db349843e6526e93cdfe6 *zxp-linux-arm64" | sha256sum  --check
zxp-linux-arm64: OK

echo "a39bc62c6175b85d9610781bd8811fa13473c9b5f32db349843e6526e93cdfe6 zxp-linux-arm64" | sha256sum  --check
zxp-linux-arm64: OK
```
On MacOS:
```
echo "a39bc62c6175b85d9610781bd8811fa13473c9b5f32db349843e6526e93cdfe6 *zxp-linux-arm64" | shasum -a 256 --check
zxp-linux-arm64: OK

echo "a39bc62c6175b85d9610781bd8811fa13473c9b5f32db349843e6526e93cdfe6 zxp-linux-arm64" | shasum -a 256 --check
shasum: standard input: no properly formatted SHA checksum lines found
```

Reply:
Thank you for the explanation